xml/soc-operations-self-sign-cert.xml

```diff
@@ -0,0 +1,784 @@
+        <imageobject role="fo">
+         <imagedata fileref="database-barclamp.png" width="100%" format="png"/>
+        </imageobject>
+        <imageobject role="html">
+         <imagedata fileref="database-barclamp.png" width="100%" format="png"/>
+        </imageobject>
+       </mediaobject>
+      </figure>
+    </step>
+    <step>
+      <para>
+        Click <guilabel>Apply</guilabel>. Your changes will apply in a
+        few minutes.
+      </para>
+    </step>
+    <step>
+      <para>
+        Set your Certificate File and Key File to the proper location, and
+        set the CA Certificates File to the <filename>/etc/ssl/ca-bundle.pem</filename>
+        in the distribution. Make sure <literal>Generate (self-signed) certificates</literal>
+        is false, and <literal>Certificate is insecure</literal> is false.
+      </para>
+      <figure>
+       <title>RabbitMQ Barclamp</title>
+       <mediaobject>
+        <imageobject role="fo">
+         <imagedata fileref="rabbitmq-barclamp.png" width="100%" format="png"/>
+        </imageobject>
+        <imageobject role="html">
+         <imagedata fileref="rabbitmq-barclamp.png" width="100%" format="png"/>
+        </imageobject>
+       </mediaobject>
+      </figure>
+    </step>
+    <step>
+      <para>
+        Click <guilabel>Apply</guilabel>. Your changes will apply in a
+        few minutes.
+      </para>
+    </step>
+    <step>
+      <para>
+        Set your Certificate File and Key File to the proper location, and
+        set the CA Certificates File to the <filename>/etc/ssl/ca-bundle.pem</filename>
+        in the distribution. Make sure <literal>Generate (self-signed) certificates</literal>
+        is false, and <literal>Certificate is insecure</literal> is false.
+      </para>
+      <figure>
+       <title>Keystone Barclamp</title>
+       <mediaobject>
+        <imageobject role="fo">
+         <imagedata fileref="keystone-barclamp.png" width="100%" format="png"/>
+        </imageobject>
+        <imageobject role="html">
+         <imagedata fileref="keystone-barclamp.png" width="100%" format="png"/>
+        </imageobject>
+       </mediaobject>
+      </figure>
+    </step>
+    <step>
+      <para>
+        Click <guilabel>Apply</guilabel>. Your changes will apply in a
+        few minutes.
+      </para>
+    </step>
+    <step>
+      <para>
+        Set your Certificate File and Key File to the proper location, and
+        set the CA Certificates File to the <filename>/etc/ssl/ca-bundle.pem</filename>
+        in the distribution. Make sure <literal>Generate (self-signed) certificates</literal>
+        is false, and <literal>Certificate is insecure</literal> is false, and
+        <literal>Require Client Certificates</literal> is true.
+      </para>
+      <figure>
+       <title>Glance Barclamp</title>
+       <mediaobject>
+        <imageobject role="fo">
+         <imagedata fileref="glance-barclamp.png" width="100%" format="png"/>
```

Comment:
for some now unknown reason, I've always set the role=fo width to 75%. Not sure who told me, Stefan or Frank. Or maybe no one, and I just saw that somewhere else and copied it. I've thought that it had to do with PDF rendering issues.

Reply:
Ohhh I think you're right about the PDFs. I recently had that problem. I admittedly copied this from another guide, I'll check in with Frank/Stefan today and see what's appropriate. Otherwise, I'll put this down to 75% for now.